README.md

```diff
@@ -1 +1,62 @@
-# recipe-app
\ No newline at end of file
+# Recipe App
+
+The Recipe app keeps track of all your recipes, ingredients, and inventory. It will allow you to save ingredients, keep track of what you have,
+
+![recipe](https://user-images.githubusercontent.com/9049260/153709727-4938019b-21cb-404e-b443-8df7abaa5e1d.png)
+## Getting Started
+
+To get a local copy up and running follow these simple example steps.
+
+## Prerequisites
+  - Ruby
+  - Rails
+  - PostgreSQL
+
+## Setup
+
+- Make sure you have Ruby IRB installed on your computer
+- Clone or download this repo on your machine
+- Enter project directory
+## Install
+
+```sh
+bundle install
+```
+
+### Database
+
+```sh
+# Create the database
+rake db:create
+
+# Load the schema
+rails db:schema:load
+
+## Apply migration (only if you generate a new migration file)
+rails db:migrate
+```
+### Run
+
+```sh
+rails s
+```
```

Comment:
[OPTIONAL]

- I would highly recommend adding the step on how to run the test as well so that it will be easier for users to test your project. [ Hint: `rspec`]

Reply:
Thank you so much. I will do it